src/applications/coronavirus-vaccination/containers/CoronavirusVaccinationApp.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+function CoronavirusVaccinationApp({ user: _user, children }) {
+  return (
+    <div className="usa-grid usa-grid-full">
+      <div className="usa-content vads-u-margin-bottom--3">
+        <h1>Request a COVID-19 vaccination</h1>
+        {children}
+      </div>
+    </div>
+  );
+}
```

Comment:
This gets us access to the user data. I'm assuming we want to use that to prefill the form fields or something more advanced.

Reply:
@ncksllvn Where can I see what data about the user we get back from `state.user`?

Reply:
Echoing from Slack -

There are more sophisticated ways (like using Redux dev tools and running the API locally to see the full live structure) but this file here gives you an idea of the data shape -> https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/user/profile/utilities/index.js#L41